config/kubernetes/sig-docs/teams.yaml

```diff
@@ -313,15 +303,32 @@ teams:
     - zhangxiaoyu-zidif
     - zparnold
     privacy: closed
-  sig-docs-vi-owners:
-    description: Admins for Vietnamese content
-    members:
-    - ngtuna
-    - truongnh1992
-    privacy: closed
-  sig-docs-vi-reviews:
-    description: PR reviews for Vietnamese content
+  website-write:
```

Comment:
```suggestion
  website-maintainers:
```

Reply:
You'll need to place the renamed team above `website-milestone-maintainers` list as well.

Reply:
Should the list not be alphabetical? Because right now teams are in alphabetical order:
> 309: website-admins
> 316: website-milestone-maintainers
> 372: website-write

Reply:
This still needs to be `website-maintainers` instead of `website-write`.